application/features/bootstrap/FeatureContext.php

```diff
@@ -590,6 +612,24 @@ public function getResponseProperty($property)
         return $this->resBody[$property];
     }
 
+    /**
+     * @param $property
+     * @return mixed
+     */
+    public function setRequestBody(string $key, string $value)
```

Comment:
You'll probably want to change this to allow `$value` to be anything, since it will automatically be JSON-encoded later and needs to still be an object at this point for WebAuthn values.
```suggestion
    public function setRequestBody(string $key, $value)
```